modules/mobile/app/services/mobile/v2/appointments/proxy.rb

```diff
@@ -35,10 +35,10 @@ def merge_clinics(appointments)
             cached_clinics[clinic_id] = get_clinic(appt[:location_id], clinic_id) unless cached
 
             service_name = cached_clinics.dig(clinic_id, :service_name)
-            appt[:service_name] = service_name if service_name
+            appt[:service_name] = service_name
```

Comment:
these are new fields in the hash. Don't want to change the schema of data going into the adapter based on the backfill upstream services. So now the field will be there regardless but may be nil.

Reply:
Ignore my comment below. I didn't see this.